src/main/resources/templates/oidcRedirect.html

```diff
@@ -26,12 +26,10 @@
 <body>
 
 <script th:inline="javascript">
-var state = [[${state}]];
-var nonce = [[${nonce}]];
-var target = [[${initiation_dto.ltiStorageTarget}]];
-window.onload = function () {
-    postStateAndNonce(state, nonce, target);
```

Comment:
Why are we removing this?  We need to post the state and nonce on this page, and I can't see the button to do it manually (maybe I'm missing it launched on another place)

Reply:
I moved this functionality here: https://github.com/Unicon/tool13demo/blob/d29a536401c8e48f58527f5b6283ca9545c78902/src/main/resources/static/js/posts.js#L142

Since any variables defined in oidcRedirect.html are globally available to all scripts on the page, we don't need to pass them as parameters to the functions, since the functions already have access to them.

Reply:
Sounds good!